GPL/DemanglerGnu/src/demangler_gnu_v2_41/c/dyn-string.c

```diff
@@ -51,7 +51,7 @@ Boston, MA 02110-1301, USA.  */
 
 /* Performs in-place initialization of a dyn_string struct.  This
    function can be used with a dyn_string struct on the stack or
-   embedded in another object.  The contents of of the string itself
+   embedded in another object.  The contents of the string itself
```

Comment:
Let's not modify this GPL-licensed file.  There's supposed to be additional bookkeeping that goes along with any changes to it.